static/app/views/replays/detail/console/index.tsx

```diff
@@ -47,8 +51,14 @@ function Console({breadcrumbs, startTimestampMs = 0}: Props) {
         })
       : undefined;
 
+  const currentUserAction = getPrevBreadcrumb({
+    crumbs: breadcrumbs,
+    targetTimestampMs: startTimestampMs + currentTime,
+    allowExact: true,
+  });
+
   return (
-    <Fragment>
+    <ConsoleContainer ref={containerRef}>
```

Comment:
Can we move the filters stuff outside of this container? Seeing the filters all the time will be useful i think.